Afform - Move status check for login-tokens. Make the text more actionable.

ufundo opened the pull request:
Overview
----------------------------------------
Changes status notice from statement of fact, to something more actionable.

Before
----------------------------------------
![image](https://github.com/user-attachments/assets/281670a6-ad8f-4216-ba37-bae373b40716)



After
----------------------------------------
![image](https://github.com/user-attachments/assets/eaccf30c-6ffd-4366-8a7d-b87fae37832e)

Link goes to AuthX settings page, where you need to have:
![image](https://github.com/user-attachments/assets/1dd22dc7-1b2a-4fce-8d6f-8b5dff1774c5)



Technical Details
----------------------------------------
1. I just rebased onto `master` and the status notice disappeared... For some reason the Afform.get in `Tokens::getTokenForms` isn't returning my form. Seems like maybe a separate issue.. 
2. Should this notice be more fine grained now @totten ? Indeed if you only need it for login tokens, could it go unconditionally into that extension?

ufundo committed:
afform - move authx cred status check to afform_login_token

Sjord commented on ext/afform/core/Civi/Afform/StatusChecks.php:
```diff
@@ -28,8 +28,9 @@ public static function hook_civicrm_check($e) {
       $e->messages[] = new \CRM_Utils_Check_Message(
         'afform_token_authx',
-        E::ts('Email token support has been configured for %1 form(s). This requires JWT authentication, <code>authx_auto_cred</code> does not include JWT. ', [
+        E::ts('Email token support has been configured for %1 form(s). This requires JSON Web Tokens are included as an acceptable credential for Auto Login in your AuthX configuration. Please <a href="%2">review your configuration here</a>.', [
```
> This requires JSON Web Tokens are included

I had a hard time understanding this sentence. I think better would be "This requires _that_ JSON web tokens are ..."

ufundo replied:
Thanks @Sjord that's helpful - I was definitely struggling to wordsmith it!

ufundo committed:
correct error message

ufundo commented:
> Should this notice be more fine grained now @totten ? Indeed if you only need it for login tokens, could it go unconditionally into that extension?

I'm fairly sure the answer is yes - the standards tokens in Afform are bespoke "form-scoped" tokens now, which don't use this setting.

So I've moved the status check to Afform Login Token, which allows simplifying (you don't need to check whether someone has created a form to warn them - if you have Afform Login Token enabled then you want this setting.

ufundo commented on ext/afform/core/Civi/Afform/Tokens.php:
```diff
@@ -152,7 +152,7 @@ private static function generateEmailVerificationUrl(int $submissionId): string
     }
     catch (CryptoException $exception) {
       \Civi::log()->warning(
-        'Civi\Afform\LegacyTokens cannot generate tokens due to crypto exception.',
```
unrelated minor correction (looks like that class moved)

totten committed:
afform_login_token - Tweak messaging

1. Reduce the depth of the sentence's parse-tree.

2. When referencing a field from authx, use its literal label (w/canonical translation).

3. If we use this extension's title (per `info.xml`) as the message title, then
   the prose doesn't have to work so hard to set context.
